### PR TITLE
Ios autocomplete

### LINKED ISSUE #37: iOS: Update station selection UI to text field with autocomplete

### PR DESCRIPTION
**Pull request by cameronmccormack**

### CODE REVIEW

File: native/KotlinIOS/KotlinIOS/ViewController.swift [hunk 1]
```diff
@@ -23,38 +24,14 @@ class ViewController: UIViewController {
         arrivalStationLabel.text = "Arrival"
         label.text = "TrainBoard"
         
-        stations.delegate = self
-        stations.dataSource = self
+        print(stationData.count)
```
Comment (billy-yoyo) [inline native/KotlinIOS/KotlinIOS/ViewController.swift]: left over debugging print

File: native/KotlinIOS/KotlinIOS/ViewController.swift [hunk 2]
```diff
@@ -99,12 +76,14 @@ extension ViewController: ApplicationContractView {
     func setStations(stations: [Station]) {
         stationData = stations;
             
-        if (!stations.isEmpty) {
-            presenter.setDepartureStation(station: stations[0])
-            presenter.setArrivalStation(station: stations[0])
+        departureStation.optionArray = stationData.map{$0.name}
```
Comment (billy-yoyo) [inline native/KotlinIOS/KotlinIOS/ViewController.swift]: Maybe use $0.toString rather than .name to unify with Android?

**Review by billy-yoyo — CHANGES_REQUESTED** [review]

File: native/KotlinIOS/KotlinIOS/ViewController.swift (hunk 1 above)
Comment (cameronmccormack) [reply native/KotlinIOS/KotlinIOS/ViewController.swift]: deleted

File: native/KotlinIOS/KotlinIOS/ViewController.swift (hunk 2 above)
Comment (cameronmccormack) [reply native/KotlinIOS/KotlinIOS/ViewController.swift]: done using $0.description()

**Review by billy-yoyo — APPROVED** [review]

**Pull request merged**